neurodamus/ngv.py

```diff
@@ -191,22 +234,27 @@ def _show_mcd(sec):
     #        )
 
     def set_pointers(self):
-        glut_list = self._glut_list
         c = self._cellref
-        index = 0
+        # the endfeet are not included in all as they are added later.
+        # I still do not know exactly when the pointers need to be
+        # reassigned and which ones are stale. the endfeet may be already
+        # up-to-date
+        all_secs = chain(c.all, self.endfeet)
+
+        # just a safety check
```

Comment:
`assert` suggests it's a safety check, it would be more helpful to add a message to the: `assert predicate, "mismatch of of cells to glut_list"` or whatever

Reply:
it is a safety check (as I wrote just before). Anyway, done

Reply:
I'm saying that using `assert` already tells signals it's a safety check; so a more helpful comment (or, message) explains the goal of the safety check.

Reply:
got it. ok